de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5554310_sys_gh6329SetDefaultColumnsToIsCalculated.sql

```diff
@@ -0,0 +1,9 @@
+update ad_column c
+set iscalculated = 'Y'
```

Comment:
```suggestion
set iscalculated = 'Y',
	Updated=`2020-03-10 17:01:56.767453+01`,
	UpdatedBy=99
```

* use a hardcoded date string that will be the same in the SQL(-migration script) and the DB
* use AD_User_ID 99 which is generally used for "manual" migrations